src/components/apis/list-of-apis/ko/runtime/api-list-dropdown.html

```diff
@@ -21,7 +21,7 @@
                 <i class="icon icon-magnifier"></i>
             </button>
         </div>
-
+        <!-- <tag-input params="{ scope: 'apis', onChange: onTagsChange }"></tag-input> -->
```

Comment:
clean up?

Reply:
Sorry, just noticed it. Will do in the next push.